x-pack/plugins/cloud_security_posture/public/pages/benchmarks/benchmarks_table.tsx

```diff
@@ -5,60 +5,138 @@
  * 2.0.
  */
 
-import { EuiBasicTable, type EuiBasicTableColumn } from '@elastic/eui';
+import {
+  EuiLink,
+  EuiBasicTable,
+  type EuiBasicTableColumn,
+  type EuiBasicTableProps,
+  type Pagination,
+  type CriteriaWithPagination,
+} from '@elastic/eui';
 import React from 'react';
 import moment from 'moment';
+import { Link, useHistory } from 'react-router-dom';
 import { TABLE_COLUMN_HEADERS } from './translations';
-import type { CspBenchmarkIntegration } from './types';
+import type { Benchmark } from '../../../common/types';
+import { pagePathGetters } from '../../../../fleet/public';
+import { useKibana } from '../../common/hooks/use_kibana';
 
-interface BenchmarksTableProps {
-  benchmarks: CspBenchmarkIntegration[];
+interface BenchmarksTableProps
+  extends Pick<EuiBasicTableProps<Benchmark>, 'loading' | 'error' | 'noItemsMessage'>,
+    Pagination {
+  benchmarks: Benchmark[];
+  setQuery(pagination: CriteriaWithPagination<Benchmark>): void;
   'data-test-subj'?: string;
 }
 
-const BENCHMARKS_TABLE_COLUMNS: Array<EuiBasicTableColumn<CspBenchmarkIntegration>> = [
+const AgentPolicyButtonLink = ({ name, id: policyId }: { name: string; id: string }) => {
+  const { http } = useKibana().services;
+  return (
+    <EuiLink
+      href={http.basePath.prepend(pagePathGetters.policy_details({ policyId }).join(''))}
+      title={name}
+    >
+      {name}
+    </EuiLink>
+  );
+};
+
+const BENCHMARKS_TABLE_COLUMNS: Array<EuiBasicTableColumn<Benchmark>> = [
   {
-    field: 'integration_name',
-    name: TABLE_COLUMN_HEADERS.INTEGRATION_NAME,
-    dataType: 'string',
+    field: 'package_policy.name',
+    name: TABLE_COLUMN_HEADERS.BENCHMARK_RULES,
+    render: (v, r) => (
+      <Link to={`/benchmark/${r.package_policy.id}/${r.package_policy.policy_id}/rules`} title={v}>
+        {v}
+      </Link>
+    ),
+    truncateText: true,
   },
   {
-    field: 'benchmark',
-    name: TABLE_COLUMN_HEADERS.BENCHMARK,
-    dataType: 'string',
+    field: 'package_policy.rules', // TODO: add fields
```

Comment:
Please make sure there's a ticket for this

Reply:
@CohenIdo should i create one or is this tracked?